### DISCUSSION TITLE
Cloudflare 似乎开始禁止自选 IP 了？

### DISCUSSION THREAD

**Discussion by FloatSheep** [XIU2/CloudflareSpeedTest#495] [讨论]
如题

![image](https://github.com/XIU2/CloudflareSpeedTest/assets/142888681/4e85009a-a1ed-4f96-9c5c-11f13c773c53)

不知道是我操作问题还是怎么，Cloudflare 跳出来 1000 错误

按照原本的操作，是解析自选 IP 之后绑定 Worker 路由就能用的，现在会跳出来 **DNS points to prohibited IP**

**Comment by FloatSheep** [comment]
我敲，是 Cloudflare Worker 的锅

**Reply by lm317379829** [reply]
> 我敲，是 Cloudflare Worker 的锅

细说

**Reply by FloatSheep** [reply]
我这要先添加 Worker 路由，然后再解析，并且解析的时候 IP 还有要求，一些自选 IP 会出现 1000 😠👎
